__tests__/integration/one-app.spec.js

```diff
@@ -700,30 +748,72 @@ describe('Tests that require Docker setup', () => {
         });
 
         describe('child module `requiredExternals` invalid usage', () => {
-          const requiredExternalsErrorMatch = /Failed to get external react-intl from root module/;
-
           const moduleName = 'cultured-frankie';
           const version = '0.0.1';
 
-          let requiredExternalsError;
+          afterEach(() => {
+            writeModuleMap(originalModuleMap);
+          });
 
-          beforeAll(async () => {
-            requiredExternalsError = searchForNextLogMatch(requiredExternalsErrorMatch);
+          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal` - logs failure', async () => {
+            const requiredExternalsErrorMatch = /Failed to get external react-intl from root module/;
+
+            const requiredExternalsError = searchForNextLogMatch(requiredExternalsErrorMatch);
             await addModuleToModuleMap({
               moduleName,
               version,
             });
             // not ideal but need to wait for app to poll;
-
             await waitFor(5000);
-          });
 
-          afterAll(() => {
-            writeModuleMap(originalModuleMap);
+            await expect(requiredExternalsError).resolves.toMatch(requiredExternalsErrorMatch);
           });
 
-          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal`', async () => {
-            await expect(requiredExternalsError).resolves.toMatch(requiredExternalsErrorMatch);
+          test('fails to get external `react-intl` for child module as an unsupplied `requiredExternal` - Logs reverting message', async () => {
+            const revertErrorMatch = /There was an error loading module (?<moduleName>.*) at (?<url>.*). Reverting back to (?<workingModule>.*)/;
+            const requiredExternalsError = searchForNextLogMatch(revertErrorMatch);
+            await addModuleToModuleMap({
+              moduleName,
+              version,
+            });
+            // not ideal but need to wait for app to poll;
+            await waitFor(5000);
```

Comment:
Are we loading the same module `'cultured-frankie';` again? if so we should try and merge this test with the one above if you are able to get both error messages at the same time. This is to avoid overloading the integration tests and make them take longer unnecessarily

Reply:
I was only able to get one regex matcher to work at a time within a single test case. Thats why I added another test case

Reply:
What does the log look like? does it log both lines?
`/Failed to get external react-intl from root module/`
and
`/There was an error loading module (?<moduleName>.*) at (?<url>.*). Reverting back to (?<workingModule>.*)`?

If so we should try and find why the regex is not matching in one single case, loading the same module to the module map twice and that 5sec delay is expensive if we can avoid it 👍